src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
* William Woodall <notifications@github.com> [2016-08-10 15:02]:
> Given that the policy is not going to change, then I'd still opt for having the normal setup files and let users source them and/or add them to their shell rc files as they like. This behavior (of always having `/usr/share` in the RPP) is unfortunately a deal breaker for using the debian version of `rospkg` with ROS installed to `/opt`.

Can you give an example where this could break so we can find a fix for
that?